functors_task_1/student/src/impl.cpp

```diff
@@ -2,28 +2,44 @@
 
 BeerBrand BeerOrganizer::operator()() {
     if (static_cast<size_t>(BeerBrand::Max) == currentIndex)
-        currentIndex = 1;
+        currentIndex = static_cast<size_t>(BeerBrand::HoeGaarden);
 
     return static_cast<BeerBrand>(currentIndex++);
 }
 
-bool isExpensiveBeer(const BeerBrand& brand)
+bool isExpensiveBeer(BeerBrand brand)
 {
-    std::vector<BeerBrand> expensive{BeerBrand::HoeGaarden, BeerBrand::Corona};
-
-    return std::find(expensive.begin(), expensive.end(), brand) != std::end(expensive);
+    switch(brand)
+    {
+    case BeerBrand::HoeGaarden:
+    case BeerBrand::Corona:
+        return true;
```

Comment:
avoid explicit true/false in your code

Reply:
Could you please explain why?

Reply:
Doing so avoids verbosity and eliminates some opportunities for mistakes. Helps make style consistent and conventional.